### PR TITLE
P-8140: Send header including sdk version

### PR DESCRIPTION
**Pull request by jesushernandez**
Also replace eslint with biome.

### CODE REVIEW

File: src/request.ts [hunk 1]
```diff
@@ -29,6 +30,7 @@ export async function request<Query, Variables>(
     const headers = {
       Authorization: `Bearer ${ctx.apiKey}`,
       'Content-Type': 'application/json',
+      'X-Plain-SDK-Version': packageJson.version,
```
Comment (mattvagni) [inline src/request.ts:33]: Is it worth adding 'typescript-sdk' somewhere in the value or name so we can support other sdks in future?

File: biome.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      "arrowParentheses": "always",
+      "bracketSpacing": true,
+      "bracketSameLine": false,
+      "quoteStyle": "single",
+      "attributePosition": "auto"
+    }
+  },
+  "files": {
+    "include": ["vitest.*.js", "src/**/*.ts", "src/**/*.gql"],
+    "ignore": ["dist/**", "node_modules/**", "src/graphql/types.ts"]
```
Comment (mattvagni) [inline biome.json:27]: should we not also ignore the webhook schema?

**Review by DLeyland — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -12,15 +12,17 @@
     "codegen:graphql": "graphql-codegen",
     "codegen:webhooks": "sh ./scripts/codegen-webhooks.sh",
     "typecheck": "tsc --noEmit",
-    "lint": "pnpm run lint:eslint && pnpm run lint:prettier",
-    "lint:eslint": "eslint 'src/**/*.ts'",
-    "lint:prettier": "prettier --config=.prettierrc --ignore-path=.prettierignore --check '**/*.{ts,tsx,gql,js}'",
+    "lint": "biome lint .",
+    "lint:fix": "biome lint . --fix",
+    "format": "biome check .",
```
Comment (mattvagni) [inline package.json:17]: check runs lint, as far as I know this means lint is superfluous - you can get rid of it.

File: .changeset/shaggy-turkeys-play.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@team-plain/typescript-sdk': patch
+---
+
+Include SDK version as header in all requests
```
Comment (mattvagni) [inline .changeset/shaggy-turkeys-play.md:5]:
```suggestion
Include SDK version as header in all requests to our API so we can better debug failures and issues.
```

**Review by mattvagni — APPROVED** [review]

**Pull request merged**